document/CoreFormat.mdk

```diff
@@ -198,50 +198,50 @@ each of which represents a single result produced in the course of a run.
 
 The `results` array shall be empty if the run that produced the `runLog` object did not produce any results.
 
-## `toolInfo` object { #toolInfo }
+## `tool` object { #tool }
 
-### General { #toolInfo-General }
+### General { #tool-General }
 
-A `toolInfo` object contains information describing the static analysis tool that was run.
+A `tool` object contains information describing the static analysis tool that was run.
 
 ~ IsoNote
 If another tool post-processes the log file (for example, by removing certain results,
 or by adding information that was not known to the static analysis tool),
-the post-processing tool should not alter any part of the `toolInfo` object.
+the post-processing tool should not alter any part of the `tool` object.
 ~
 
 ~ IsoExample
 
 ```
 {
-    "name": "CodeScanner",                                       # see \(&sect;[#toolInfo-name]\)
-    "fullName": "CodeScanner 2.0, Developer Preview (en-US)",    # see \(&sect;[#toolInfo-fullName]\)
-    "version": "2.0.0-beta",                                     # see \(&sect;[#toolInfo-version]\)
-    "fileVersion": "2.0.1502.2"                                  # see \(&sect;[#toolInfo-fileVersion]\)
+    "name": "CodeScanner",                                       # see \(&sect;[#tool-name]\)
+    "fullName": "CodeScanner 2.0, Developer Preview (en-US)",    # see \(&sect;[#tool-fullName]\)
+    "version": "2.0.0-beta",                                     # see \(&sect;[#tool-version]\)
+    "fileVersion": "2.0.1502.2"                                  # see \(&sect;[#tool-fileVersion]\)
 }
 ```
 ~
 
-### `name` property { #toolInfo-name }
+### `name` property { #tool-name }
 
-A `toolInfo` object shall contain a property named `name` whose value is a string containing the name of the tool that produced the log file.
+A `tool` object shall contain a property named `name` whose value is a string containing the name of the tool that produced the log file.
 
 ~ IsoExample
 `"CodeScanner"`
 ~
 
-### `fullName` property { #toolInfo-fullName }
+### `fullName` property { #tool-fullName }
 
-A `toolInfo` object may contain a property named `fullName` whose value is a string containing the name of the tool
+A `tool` object may contain a property named `fullName` whose value is a string containing the name of the tool
 along with its version and any other useful identifying information, such as its locale.
 
 ~ IsoExample
 `"CodeScanner 2.0, Developer Preview (en-US)"`
 ~
 
-### `version` property { #toolInfo-version }
+### `version` property { #tool-version }
 
-A `toolInfo` object should contain a property named `version` whose value is a string containing the tool version.
+A `tool` object should contain a property named `version` whose value is a string containing the tool version.
 If present, the string should be in the format specified by Semantic Versioning 2.0.0.
```

Comment:
I'd like to get rid of this passage where we seem to be telling tool vendors how to version their tools. I'd delete lines 245-253.